libresapi/src/api/TransfersHandler.h

```diff
@@ -5,16 +5,20 @@
 
 #include <retroshare/rsfiles.h>
 #include <retroshare/rspeers.h>
+#include <retroshare/rsnotify.h>
 
 namespace resource_api
 {
 
-class TransfersHandler: public ResourceRouter, Tickable
+class TransfersHandler: public ResourceRouter, Tickable, NotifyClient
 {
 public:
-	TransfersHandler(StateTokenServer* sts, RsFiles* files, RsPeers *peers);
+	TransfersHandler(StateTokenServer* sts, RsFiles* files, RsPeers *peers, RsNotify* notify);
     virtual ~TransfersHandler();
 
+	// from NotifyClient

```

Comment:
Please use Doxigen style comments for headers documentation like https://github.com/RetroShare/RetroShare/blob/master/libretroshare/src/gxstrans/p3gxstrans.h#L210